src/meta/src/stream/sink.rs

```diff
@@ -24,8 +24,12 @@ pub async fn validate_sink(
     connector_rpc_endpoint: Option<String>,
 ) -> MetaResult<()> {
     let sink_catalog = SinkCatalog::from(prost_sink_catalog);
-    let sink_config = SinkConfig::from_hashmap(sink_catalog.properties.clone())
+    let mut properties = sink_catalog.properties.clone();
+    // Insert a value as the `identifier` field to get parsed by serde.
```

Comment:
What does this line mean?

Reply:
The `identifier` field is supposed to be inserted when we create the sink executor. We insert a dummy value here in meta to ensure a successful parsing. (The field is originally defined here: https://github.com/risingwavelabs/risingwave/blob/main/src/connector/src/sink/kafka.rs#L66)